cmd/tk/util.go

```diff
@@ -15,31 +15,36 @@ func pageln(i ...interface{}) {
 	fPageln(strings.NewReader(fmt.Sprint(i...)))
 }
 
-// fPageln invokes the systems pager with the supplied data
-// falls back to fmt.Println() when paging fails or non-interactive
+// fPageln invokes the systems pager with the supplied data.
+// If the PAGER environment variable is empty, no pager is used.
+// If the PAGER environment variable is unset, use less with posix flags.
 func fPageln(r io.Reader) {
-	// get system pager, fallback to `less`
-	pager := os.Getenv("PAGER")
-	var args []string
-	if pager == "" || pager == "less" {
+	pagerEnv, ok := os.LookupEnv("PAGER")
+	if !ok {
 		// --RAW-CONTROL-CHARS  Honors colors from diff. Must be in all caps, otherwise display issues occur.
 		// --quit-if-one-screen Closer to the git experience.
 		// --no-init            Don't clear the screen when exiting.
-		pager = "less"
-		args = []string{"--RAW-CONTROL-CHARS", "--quit-if-one-screen", "--no-init"}
+		pagerEnv = "less --RAW-CONTROL-CHARS --quit-if-one-screen--no-init"
 	}
 
-	// invoke pager
-	cmd := exec.Command(pager, args...)
-	cmd.Stdin = r
-	cmd.Stdout = os.Stdout
-	cmd.Stderr = os.Stderr
+	pager := strings.Split(pagerEnv, " ")
```

Comment:
This is whitespace sensitive, right?

Either use [`strings.Fields`](https://golang.org/pkg/strings/#Fields), but why not go for a full shell, invoking whatever you get from the env as `sh -c`?

Reply:
Yeah, this is really messy and as you saw below, I didn't even guard the panic properly :)

I've pushed some changes that use `sh -c` and it is much cleaner.

Reply:
Looks really good now!